Engine/system/api/x11api.cpp

```diff
@@ -180,6 +180,7 @@ SystemApi::Window *X11Api::implCreateWindow(Tempest::Window *owner, uint32_t w,
   auto ret = reinterpret_cast<SystemApi::Window*>(win.ptr());
   windows[ret] = owner;
   //maximizeWindow(win);
+  setAsFullscreen(ret, true); // This had to be added for the setAsFullscreen function to be called for me
```

Comment:
As mention in comment to PR - this line is look correct.
implCreateWindow(Tempest::Window *owner, uint32_t w, uint32_t h) suppose to create a `Normal` window, not a full-screen one.